util/screenshots.js

```diff
@@ -0,0 +1,84 @@
+import fs from "fs";
+import path from "path";
+import * as warcio from "warcio";
+
+
+// ============================================================================
+
+class Screenshots {
+
+  constructor({page, url, date, directory}) {
+    this.page = page;
+    this.url = url;
+    this.directory = directory;
+    this.warcName = path.join(this.directory, "screenshots.warc.gz");
+    this.date = date ? date : new Date();
+
+    this.screenshotOptions = {
```

Comment:
Can we make this an exported array / json blob and just check for the config that way?

eg.

```
export const screenshotTypes = {
"view": ...,
"thumbnail": ...
}
```
and then just check if the passed in values are one of these in params validation